src/color/readableColor.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import parseToRgb from './parseToRgb'
+import curry from '../internalHelpers/_curry'
+
+/**
+ * Selects black or white or best contrast depending on the luminosity of the supplied color.
+ * Follows W3C specs for readability
+ *
+ * @example
+ * // Styles as object usage
+ * const styles = {
+ *   color: mostReadable('#000'),
+ *   color: mostReadable('rgb(255,0,0)'),
+ * }
+ *
+ * // styled-components usage
+ * const div = styled.div`
+ *   color: ${mostReadable('#000')};
+ *   color: ${mostReadable('rgb(255,0,0)')};
+ * `
+ *
+ * // CSS in JS Output
+ *
+ * element {
+ *   color: "#fff";
+ *   color: "#000";
+ * }
+ */
+
+function readableColor(color: string): string {
+  const w3cConversion = (rgbColor: number): number => {
```

Comment:
you can move it outside and before like normal. Just move the documentation block so it is just before the function you are documenting.